modules/gateways/blockonomics.php

```diff
@@ -91,6 +98,16 @@ function reqListener () {
 				return false;
 			}
 
+			/**
+			 * Prompt to save changes after setting a new API key 
+			 */
+			var apiKeyField = document.getElementsByName('field[ApiKey]')[0];
+			apiKeyField.onchange = function() {
+				newBtn.disabled = true;
```

Comment:
Is this disabling Test Setup ? When does this btn get enabled again ? 
Prefer not to disable things unless very necessary

Reply:
This is functionally the same as showing error message on JS alert() and not allowing to continue
https://screenshots.firefox.com/Z8wUQjhdFmoGFDQ3/92.222.83.165
The Test Setup button will enable once saved/refreshed but will not re-enable if API key field is restored to original value.
Can be left as is or set as other plugins

Reply:
I prefer not to disable the button. Creates UI issues. We can remove this